README.md

```diff
@@ -1,30 +1,96 @@
-## Synopsis
+# FreeOrion
 
-Source code repository of the FreeOrion project.
+FreeOrion is a free, Open Source, turn-based space empire and galactic conquest
+computer game being designed and built by the FreeOrion project.
 
-FreeOrion is a free, open source, turn-based space empire and galactic conquest (4X) computer game being designed and built by the FreeOrion project. FreeOrion is inspired by the tradition of the Master of Orion games, but is not a clone or remake of that series or any other game. 
+Freeorion builds on the classic *4X* model by incorporating the nation-building
+elements of games such as *Europa Universalis 2*.  While its modular,
+Open Source design allows for a significant degree of customization of the game
+engine and the story elements by the community, the FreeOrion project is
+dedicated to the construction of a living, breathing universe in a
+*grand campaign* model.
 
-## Vision Statement
 
-FreeOrion, an open-source game inspired by Master of Orion, is a turn-based game of epic space strategy that builds on the classic '4X' model by incorporating the nation-building elements of games such as Europa Universalis 2 and a versatile tactical combat engine. While its modular, open-source design allows for a significant degree of customization of the game engine and the story elements by the community, the FreeOrion team is dedicated to the construction of a living, breathing universe in a 'grand campaign' model.
+## Requirements
 
-## How to contribute
+FreeOrion requires an *OpenGL 2.1* capable graphic card and a display with a
+minimum resolution of at least *800x600*.
 
-For more details please see our [Contribution Guidelines](CONTRIBUTING.md).
+FreeOrion requires at least *Windows XP* with *Service Pack 2* or later,
+*Mac OSX 10.7* or later or any reasonable recent Linux distribution on x86
```

Comment:
Actually, I'm currently seriously considering updating the minimum OSX version required to 10.9. After the switch to supporting C++11, I haven't been able to successfully build and run FO on OSX 10.8 or earlier. 10.9+ works fine.

And I don't see myself having time to track down and fix that issue anytime soon. The issue looks tricky, tracking down and fixing it tedious and time-comsuming. Considering that the user base playing FO on OSX versions prior to 10.9 is probably very small (if there are any left at all), I don't think it's worth the effort.

Although if, we have to correct that everywhere, not just here of course. Just giving a fair warning here that this change is very likely going to happen soon (as in, before the release).